src/pages/contact/contact-form-section.tsx

```diff
@@ -30,9 +20,34 @@ class ContactFormSection extends React.Component<IProps, IState> {
     };
   }
 
+  formValid = () => {
+    let formErrors = this.state.formErrors;
+    let name = this.state.name;
+    let email = this.state.email;
+    let message = this.state.message;
+    let valid = true;
+
+    if(name && name.length < 10){
+      formErrors.name = 'Name is required';
+      valid = false;
+    }
+    if(email && !emailRegEX.test(email)){
+      formErrors.email = 'Invalid email address';
+      valid = false;
+    }
+    if(message && message.length < 10){
+      formErrors.message = 'Write a message to us / Minimum 10 Characters required'
```

Comment:
Change `'Write a message to us / Minimum 10 Characters required'` to `'Message must have at least 10 characters'`. And add another validation to check whether the message field is empty or not.
```
if (!message) {
   formErrors.message = 'Message is required';
   valid = false;
}
```